tests/resources/mlflow-test-plugin/mlflow_test_plugin/fake_deployment_plugin.py

```diff
@@ -27,6 +27,10 @@ def get_deployment(self, name):
     def predict(self, deployment_name, df):
         return "1"
 
+    def explain(self, deployment_name, df):
+        # pylint: disable=unused-argument
```

Comment:
Since explain method is not availabe in basedeployment client, pylint throws unused argument warning. Ignoring the warning.

Reply:
I think if we want to add this API to the MLflow core CLI, we'll need to add it to the base deployment client - it's a bit strange from a user perspective for this functionality to be CLI-only, as it's then not possible to write e.g. a Python script to perform deployment and all other deploy plugin functionality is available via both the API and CLI

Reply:
If we declare the explain method as abstract method in base deployment class then client instance creation for other flavour's will fail if the abstract method is not implemented in those plugin classes. Please refer to the attached snapshots. Can we add explain as a non-abstract method in base deployment class so that it won't break the other flavour's instance creation?

explain method as abstract method.
![Screenshot from 2021-05-31 17-43-29](https://user-images.githubusercontent.com/83957636/120209431-4ec88800-c24c-11eb-9d93-bddfe3a0a729.png)

explain as non-abstract method
![Screenshot from 2021-05-31 17-45-57](https://user-images.githubusercontent.com/83957636/120209591-7e779000-c24c-11eb-8059-dac01eacb869.png)